libraries/chain/database.cpp

```diff
@@ -284,14 +285,17 @@ namespace golos { namespace chain {
                         }
 
                         auto end = fc::time_point::now();
+                        auto cur_block_pos = _block_log.get_block_pos(cur_block_num);
                         auto cur_block = *_block_log.read_block_by_num(cur_block_num);
 
-                        if (cur_block_num % 100000 == 0) {
+                        if (cur_block_pos - last_logged_block_pos >= 100000000) {
                             std::cerr
-                                << "   " << double(cur_block_num * 100) / last_block_num << "%   "
-                                << cur_block_num << " of " << last_block_num
+                                << "   " << double(cur_block_pos * 100) / last_block_pos << "%   "
+                                << cur_block_pos << " of " << last_block_pos
```

Comment:
Return where - to calculation or to "... of ..."? Could write more accurate comments and issue texts?

Reply:
% from position is good, and it is already shows progress in bytes.
`cur_block_pos << " of " << last_block_pos` is not informal, because there will be very big numbers and it is difficult to compare them
so, % from position, but `cur_block_num << " of " << last_block_num`